integ/proceed_on_state_start_fail_workflow_state1.go

```diff
@@ -0,0 +1,38 @@
+package integ
+
+import (
+	"errors"
+
+	"github.com/indeedeng/iwf-golang-sdk/gen/iwfidl"
+	"github.com/indeedeng/iwf-golang-sdk/iwf"
+)
+
+type proceedOnStateStartFailWorkflowState1 struct {
+	output string
+}
+
+func (b *proceedOnStateStartFailWorkflowState1) GetStateId() string {
+	return "proceed_on_state_start_fail_workflow_state1"
+}
+
+func (b *proceedOnStateStartFailWorkflowState1) Start(ctx iwf.WorkflowContext, input iwf.Object, persistence iwf.Persistence, communication iwf.Communication) (*iwf.CommandRequest, error) {
+	var i string
+	input.Get(&i)
+	b.output = i + "_state1_start"
```

Comment:
do we need `i +`?

Reply:
yes, input string added with _state1_start and other strings for recording the api being called